manifests/mlops/core-modules.yaml

```diff
@@ -1,146 +0,0 @@
-name: eks
```

Comment:
I think Kubernetes-based mlops are still in ADDF.

Reply:
the kf-users and kf-platform manifests (backed by `modules/mlops/kf*`) should be deprecated ...  Kubeflow manifests are no longer supported by DoEKS...so we need to remove the modules from the next release and tidy up the manifests...
@malachi-constant @srinivasreddych @kukushking we should get together and create an action item...will only take 10 minutes